VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsBaseFragment.java

```diff
@@ -63,10 +63,9 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 DiscussionThread thread = discussionPostsAdapter.getItem(position);
                 router.showCourseDiscussionResponses(context, thread, courseData);
 
-                new SetThreadReadTask(context, thread, true).execute();
                 if (!thread.isRead()) {
                     // Refresh the row to mark it as read immediately,
-                    // pending the response from the server.
+                    // Read state will update when we return to the current activity. (silent refresh)
```

Comment:
Read state has already been updated 😉 Maybe replace with 'There will be a silent refresh upon return to this Activity.'